Lib/test/test_exceptions.py

```diff
@@ -671,13 +671,20 @@ def test_str(self):
         self.assertTrue(str(Exception('a')))
         self.assertTrue(str(Exception('a', 'b')))
 
-    def testExceptionCleanupNames(self):
+    def test_exception_cleanup_names(self):
         # Make sure the local variable bound to the exception instance by
         # an "except" statement is only visible inside the except block.
         try:
             raise Exception()
         except Exception as e:
```

Comment:
One more nit: let's use a different name other than `e`
Because the second `except` uses it as well.

But, we test `locals()`, so it might be sensetive.

Reply:
Good point: we should use different names in case there's some wacky compiler interaction. Maybe e1 and e2.

Reply:
Might also be worth doing something like `with assertRaises(UnboundLocalError): e` in case the way `locals()` is created is somehow different from accessing local variables directly.

Reply:
> Good point: we should use different names in case there's some wacky compiler interaction. Maybe e1 and e2.

Or split it into two test functions.

Reply:
Two functions, added UnboundLocalError check as well.